ADKP/dca_config.json

```diff
@@ -17,15 +17,19 @@
     "sidebar_col": "#191919"
   },
   "schematic": {
+    "global": {
+      "data_model_labels": "class_Label"
```

Comment:
Capital L needs to be lowercased.

Reply:
Good catch! Spelling, what is it 🤦🏻‍♀️ Fixed.